scipy/stats/_continuous_distns.py

```diff
@@ -207,6 +208,8 @@ class alpha_gen(rv_continuous):
     %(example)s
 
     """
+    _support_mask = rv_continuous._open_support_mask
```

Comment:
Style nitpick (non-blocking): IMO it's cleaner to only have a single method `_support_mask` in `rv_continuous` and override it explicitly in the relevant distributions.